command/connect/envoy/envoy.go

```diff
@@ -493,14 +494,24 @@ func (c *cmd) templateArgs() (*BootstrapTplArgs, error) {
 		adminAccessLogPath = DefaultAdminAccessLogPath
 	}
 
+	var ca []byte
+	if httpCfg.TLSConfig.CAFile != "" {
```

Comment:
I noticed that we don't have CAPath support and I wonder if envoy can deal with multiple CAs and if we should add it.